Backend/.env

```diff
@@ -0,0 +1,2 @@
+PORT=6969
```

Comment:
Add `.env` to `.gitignore` and remove from here, it should not be on github.

Reply:
Okay, will do